app/client-v2/playwright.config.ts

```diff
@@ -43,26 +43,33 @@ export default defineConfig({
     headless: !!process.env.CI
   },
 
-  /* Configure projects for major browsers */
+  /* Configure projects for chrome. Slow running tests thus not worth running on all browsers*/
```

Comment:
hmmm i think there is value in running these tests on all browsers, often times these tests dont only catch actual bugs (usually unit/integration tests are good at catching them) but they also can catch when youve broken stuff on safari or something cos who uses safari :laughing: 

if speed is an issue then i think making these run in parallel should do the trick

Reply:
Could potentially do an all-browsers tests nightly rather than on every push if worried about speediness - but we'd need to be sure we'd notice the failure!